module/VuFindTheme/src/VuFindTheme/ThemeInfo.php

```diff
@@ -176,7 +197,7 @@ protected function loadThemeConfig($theme)
     public function getThemeInfo()
     {
         // Fill in the theme info cache if it is not already populated:
-        if (null === $this->allThemeInfo) {
+        if ($this->allThemeInfo === null) {
```

Comment:
Any particular reason you reversed the comparison here? Seems like a potentially unnecessary diff.

Reply:
I thought the reversed form was preferred since it prevents accidental assignments, but it's definitely not a hill I'm willing to die on. ;-)